### PR TITLE
Global Seed Bug Fix

### PR DESCRIPTION
**Pull request by jessechancy**
Changed `tf.random.set_seed()` to `tf.keras.utils.set_random_seed(42)`

### CODE REVIEW

**Comment by jessechancy** [comment]
Decided to remove seeded generation tests due to flakiness and inconsistent behavior on different machines

**Review by chenmoneygithub — APPROVED** [review]
